holidayextras/cache-wrapper
# FP-285: Fix redis connection drop

#### What does this PR do? (please provide any background)
[FP-285](https://hxshortbreaks.atlassian.net/browse/FP-285)

updates most of the packages of cache-wrapper to fix a connection drop issue with redis, replaces make-up for standardjs

#### What tests does this PR have?
none

#### How can this be tested?
see either https://github.com/holidayextras/the-works/pull/564 or https://github.com/holidayextras/revolver/pull/873

#### Any tech debt?
nope
#### Screenshots / Screencast

#### What gif best describes how you feel about this work?
![]()

- I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

By approving a review you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

---
